node/src/bootstrapper.rs

```diff
@@ -335,6 +335,7 @@ impl BootstrapperConfig {
             blockchain_bridge_config: BlockchainBridgeConfig {
                 blockchain_service_url: None,
                 chain_id: 3u8, /*DEFAULT_CHAIN_ID*/
+                //TODO this seems wrong, why do we want Ropsten to be a default chain?
```

Comment:
Yes, it's wrong for go-live on mainnet but correct for the time-being while testing. The comment is there to remind us the go-live on mainnet version line of code will be:
`chain_id: DEFAULT_CHAIN_ID,`

Reply:
This isn't as mild issue as it looks like. First it's quite hard to discover the fact that the BootstrapperConfig always contains this value initially unless it goes through further configuration. I adapted it to he mainnet now just to know and 59 unit tests in the node crate failed because of it, so it quite has an impact. Not everyone can predict this when he's writing tests, especially newcomers.
 
Another argument is that in all kinds of tests except multinode test we don't really connect to the blockchain and if we do (in the multinode tests) then we use a safe solution with a testnet. Which I'd like to stabilize with my changes from GH-473, where I want to make it clear that there is an only option for the multinode_test for the chain and I try to name constants there with clear names saying that alone. 

So my decision is, finally, I don't want to make a storm, I just want to replace this number 3 with a constant saying TEST_DEFAULT_CHAIN or something, so you know exactly and instantly. It's enough for me. But that will be settled in the other card.